### ISSUE TITLE
Is there an alternative way to scroll for android?

### ISSUE THREAD

**Issue by Fdde11** [AnyChart/AnyChart-Android#185]
Hi!
Currently, I'm using "xScroller(true)" to control the movement in my chart but the control is a bit clumsy on android, is there an alternative way to move around the X-axis? perhaps make it work like a horizontal scroller where is possible to use one's finger to scroll around?

If no, is it possible to increase the size of the "thumbs" in the scroller? I tried "xScroller().thumbs().normal().size(20)" but the thums size doesnt change.

**Comment by Shestac92** [comment]
@Fdde11 
Unfortunately, the current version of the library doesn't support adjusting the thumbs size. We will take into account for future updates.
Also, xScroller is the only tool for scrolling basic charts at the moment.

**Comment by martineboh** [comment]
Any way to programmatically scroll that xScroller to allow 1,000 points shown at a time?

**Comment by Shestac92** [comment]
@martineboh 
Yes, you can use the same approach described in [the issue](https://github.com/AnyChart/AnyChart/issues/181).
